api/permissions.py

```diff
@@ -78,6 +78,8 @@ def has_permission(self, request, view):
             except KeyError:
                 raise exceptions.ParseError("Missing required relationship: response")
 
+            request_data.pop("type")
+
```

Comment:
I wonder if [this change](https://github.com/django-json-api/django-rest-framework-json-api/blob/main/CHANGELOG.md#deprecated-1) to `djangorestframework-jsonapi` caused the need for this line. 
> Deprecated support for field name type as it may not be used according to the [JSON:API spec](https://jsonapi.org/format/#document-resource-object-fields).

In any case, it would probably make more sense to remove the type field from the Lamba function code.